### PR TITLE
mkdocs.yml: add links of social media

### PR DESCRIPTION
**Pull request by Domarys**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+#site_description: 'All your devices, always updated'
+#site_url: 'https://github.com/UpdateHub'
+
+# Repository
+repo_name: 'shellhub-io/shellhub-io.github.io'
+repo_url: 'https://github.com/shellhub-io/shellhub-io.github.io'
+
+
+# Copyright
+copyright: 'Copyright &copy; 2019 O.S.System'
```
Comment (otavio) [inline mkdocs.yml]: The company name is `O.S. Systems`

**Review by otavio — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (gustavosbarreto) [reply mkdocs.yml]: The year is 2020

Comment (Domarys) [reply mkdocs.yml]: fixed

File: mkdocs.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ repo_url: 'https://github.com/shellhub-io/shellhub-io.github.io'
 
 
 # Copyright
-copyright: 'Copyright &copy; 2019 O.S.System'
+copyright: 'Copyright &copy; 2019 O.S. System'
```
Comment (otavio) [inline mkdocs.yml]: @Domarys the other minor thing is the year. We are past 2019 already ;-)

File: mkdocs.yml [hunk 3]
```diff
@@ -36,3 +57,10 @@ markdown_extensions:
       custom_checkbox: true
   - pymdownx.tilde
   - attr_list
+
+# Navigation
+nav:
+  - What is ShellHub?: index.md
+  - Introduction: introduction.md
+  - Getting started: getting-started.md
+
```
Comment (otavio) [inline mkdocs.yml]: There is no newline at end of file. Likely previous line has a spurious space or something like that.

File: mkdocs.yml [hunk 4]
```diff
@@ -63,4 +63,4 @@ nav:
   - What is ShellHub?: index.md
   - Introduction: introduction.md
   - Getting started: getting-started.md
-  
\ No newline at end of file
+
```
Comment (otavio) [inline mkdocs.yml:66]: This commit should be squashed on previous one.

**Review by otavio — CHANGES_REQUESTED** [review]

**Pull request merged**